### PR TITLE
Added an awesome 404 not found page

### PR DESCRIPTION
**Pull request by AllUsernamesHaveBeenTaken**
page-not-found

404-on-mobile

https://github.com/rit-sse/OneRepoToRuleThemAll/issues/225

### CODE REVIEW

**Comment by billdybas** [comment]
Wow! This is neat! Thanks! Reviewing right now.

File: app/scss/page.scss [hunk 1]
```diff
@@ -30,3 +30,94 @@
     text-decoration: none;
   }
 }
+
+#notfound {
```
Comment (billdybas) [inline app/scss/page.scss]: We're actually in the process of moving our styles into [`styled-components`](https://www.styled-components.com/). Could you please convert these into components in `app/js/static/Page.jsx`?

File: app/scss/page.scss [hunk 2]
```diff
@@ -30,3 +30,94 @@
...
+  text-align: center;
+  line-height: 1.4;
+}
+
+.notfound .notfound-404 {
+  height: 190px;
+}
+
+.notfound .notfound-404 h1 {
+  font-family: 'Montserrat', sans-serif;
```
Comment (billdybas) [inline app/scss/page.scss]: All our headings should use "Minion Pro bold". See: https://github.com/rit-sse/branding#headings

File: app/js/static/Page.jsx [hunk 3]
```diff
@@ -22,7 +22,21 @@ const Static = ({ location }) => {
...
     <div className="container page">
-      <h4>Page Not Found</h4>
+      <div id="notfound">
+        <div className="notfound">
+          <div className="notfound-404">
+            <h1>
+              4<span />4
+            </h1>
+          </div>
+          <h2>Oops! Page Not Be Found</h2>
```
Comment (billdybas) [inline app/js/static/Page.jsx]: "Page Not Found"

File: app/js/static/Page.jsx [hunk 4]
```diff
@@ -22,7 +22,21 @@ const Static = ({ location }) => {
...
+          <p>
+            Sorry but the page you are looking for does not exist, have been
```
Comment (billdybas) [inline app/js/static/Page.jsx]: "Sorry! The page you were trying to reach does not exist or no longer exists."

File: app/js/static/Page.jsx [hunk 5]
```diff
@@ -22,7 +22,21 @@ const Static = ({ location }) => {
...
+            removed. name changed or is temporarily unavailable
+          </p>
+          <a href="/">Back to homepage</a>
```
Comment (billdybas) [inline app/js/static/Page.jsx]: This should be a `<Link>` component

**Comment by AllUsernamesHaveBeenTaken** [comment]
Thanks for the review @billdybas. I'll handle this tomorrow.

File: app/scss/page.scss (hunk 1 above)
Comment (AllUsernamesHaveBeenTaken) [reply app/scss/page.scss]: Can you assign this to someone else? Thanks.

Comment (billdybas) [reply app/scss/page.scss]: I'll add to your fork's branch this weekend.

**Comment by billdybas** [comment]
Desktop
screenshot of desktop view

Mobile
screenshot of mobile view

**Comment by AllUsernamesHaveBeenTaken** [comment]
Awesome!

**Pull request merged**